### PR TITLE
refactor(presenter): improve naming conventions and folder structures

### LINKED ISSUE #156: 🔧 [Refactor] Improve naming conventions and folder structures
## 🔧 Refactor

Improve naming conventions and folder structures

This mainly applies to the presenter, but certainly a quick look on the backend can't hurt.

Things that come to mind are the section namings, which cause unordered import statements:
image

but also things like `LogoImage` residing in `menu`:
image

The idea of this issue is to figure out how we could improve the folder structure so that its more intuitive or easier to use and find content.

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest
[See name and structure changees list](https://github.com/dreammall-earth/dreammall.earth/issues/156#issuecomment-1896217122)

### Issues
- fixes #156 
- relates #140 

### Todo

- [X] None

### CODE REVIEW

File: presenter/src/components/sections/AboutSection.stories.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import { SBComp } from '#types/SBComp'
 
-import SecondSection from './SecondSection.vue'
+import AboutSection from './AboutSection.vue'
 
 import type { Meta, StoryObj } from '@storybook/vue3'
 
 // More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
 const meta = {
   title: 'Sections/Section2',
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/AboutSection.stories.ts:9]: Adjust name

`title: 'Sections/About'`

File: presenter/src/components/sections/CallToActionSection.stories.ts [hunk 2]
```diff
@@ -1,18 +1,18 @@
 import { SBComp } from '#types/SBComp'
 
-import SixthSection from './SixthSection.vue'
+import CallToActionSection from './CallToActionSection.vue'
 
 import type { Meta, StoryObj } from '@storybook/vue3'
 
 // More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
 const meta = {
   title: 'Sections/Section6',
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/CallToActionSection.stories.ts:9]: same

File: presenter/src/components/sections/CoffeeSection.stories.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
 import { SBComp } from '#types/SBComp'
 
-import FifthSection from './FifthSection.vue'
+import CoffeeSection from './CoffeeSection.vue'
 
 import type { Meta, StoryObj } from '@storybook/vue3'
 
 // More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
 const meta = {
   title: 'Sections/Section5',
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/CoffeeSection.stories.ts:9]: same

File: presenter/src/components/sections/IntroSection.stories.ts [hunk 4]
```diff
@@ -1,18 +1,18 @@
 import { SBComp } from '#types/SBComp'
 
-import FirstSection from './FirstSection.vue'
+import IntroSection from './IntroSection.vue'
 
 import type { Meta, StoryObj } from '@storybook/vue3'
 
 // More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
 const meta = {
   title: 'Sections/Section1',
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/IntroSection.stories.ts:9]: same

File: presenter/src/components/sections/PlanningSection.stories.ts [hunk 5]
```diff
@@ -1,18 +1,18 @@
 import { SBComp } from '#types/SBComp'
 
-import FourthSection from './FourthSection.vue'
+import PlanningSection from './PlanningSection.vue'
 
 import type { Meta, StoryObj } from '@storybook/vue3'
 
 // More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
 const meta = {
   title: 'Sections/Section4',
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/PlanningSection.stories.ts:9]: same

File: presenter/src/components/sections/ProductsSection.stories.ts [hunk 6]
```diff
@@ -1,18 +1,18 @@
 import { SBComp } from '#types/SBComp'
 
-import ThirdSection from './ThirdSection.vue'
+import ProductsSection from './ProductsSection.vue'
 
 import type { Meta, StoryObj } from '@storybook/vue3'
 
 // More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
 const meta = {
   title: 'Sections/Section3',
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/ProductsSection.stories.ts:9]: same

**Review by ulfgebhardt — DISMISSED** [review]
Storybook names need to be adjusted - besides that I think this is fine, but as stated before I think @maeckes-infinitylabs should have the final word here.

File: presenter/src/components/sections/AboutSection.stories.ts (hunk 1 above)
Comment (mahula) [reply presenter/src/components/sections/AboutSection.stories.ts]: Fixed in 21ffc92bb4e09b7295f9898c72ea48319ea63f0a.

File: presenter/src/components/sections/CallToActionSection.stories.ts (hunk 2 above)
Comment (mahula) [reply presenter/src/components/sections/CallToActionSection.stories.ts]: Fixed in 21ffc92bb4e09b7295f9898c72ea48319ea63f0a.

File: presenter/src/components/sections/CoffeeSection.stories.ts (hunk 3 above)
Comment (mahula) [reply presenter/src/components/sections/CoffeeSection.stories.ts]: Fixed in 21ffc92bb4e09b7295f9898c72ea48319ea63f0a.

File: presenter/src/components/sections/PlanningSection.stories.ts (hunk 5 above)
Comment (mahula) [reply presenter/src/components/sections/PlanningSection.stories.ts]: Fixed in 21ffc92bb4e09b7295f9898c72ea48319ea63f0a.

File: presenter/src/components/sections/ProductsSection.stories.ts (hunk 6 above)
Comment (mahula) [reply presenter/src/components/sections/ProductsSection.stories.ts]: Fixed in 21ffc92bb4e09b7295f9898c72ea48319ea63f0a.

File: presenter/src/components/sections/IntroSection.stories.ts (hunk 4 above)
Comment (mahula) [reply presenter/src/components/sections/IntroSection.stories.ts]: Fixed in 21ffc92bb4e09b7295f9898c72ea48319ea63f0a.

**Comment by maeckes-infinitylabs** [comment]
image

looks good to me. but i recognized a wrong file name (see picture above).
maybe it would be good to fix it in this PR?

**Review by maeckes-infinitylabs — APPROVED** [review]

**Pull request merged**